pandas/tests/arithmetic/test_array_ops.py

```diff
@@ -34,3 +35,15 @@ def test_object_comparison_2d():
     right.flags.writeable = False
     result = comparison_op(left, right, operator.ne)
     tm.assert_numpy_array_equal(result, ~expected)
+
+
+@pytest.mark.parametrize(
+    "data, expected", [([0, 1, 2], [0, 2, 4])],
+)
+def test_extension_array_add(box_1d_array, data, expected):
```

Comment:
can you parameterize ser on Series, to_array, Index (you can just make another box parameter but doesn't need to go in conftest it can be here)

Reply:
Seems that Index and to_array do not yet support initialization from an ExtensionArray. Series dtype is correctly set to Int64, but Index and to_array have an object dtype and not an integer dtype. Do we then still want to add parametrize ser?

```
>>> import pandas as pd 
>>> import pandas._testing as tm

>>>  arr = pd.array([0, 1, 2 ], pd.Int64Dtype())

>>> pd.Series(arr)
0    0
1    1
2    2
dtype: Int64

>>> pd.Index(arr)
Index([0, 1, 2], dtype='object')

>>> tm.to_array(arr)
array([0, 1, 2], dtype=object)

Reply:
Index is correct. the to_array actually need to be fixed to simply call pd.array (try this and see if it *just* works, if it does ok), if now we can do after.

Reply:
Add parametrization to test `Index` and `tm.to_array`. 

Didnt update the `to_array` function to simply call pd.array, since multiple tests failed when I did that. Can open a PR to explore that next week.

Reply:
@jreback could you have a 2nd look? Thanks!